test/stock_lifo-stock.spec.js

```diff
@@ -0,0 +1,23 @@
+const
```

Comment:
Plik ma w nazwie underscore mimo, że przed chwilą pisałem, że jedyny dopuszczalny separator to myślnik.